test/e2e/scheduling/preemption.go

```diff
@@ -467,6 +468,71 @@ var _ = SIGDescribe("PreemptionExecutionPath", func() {
 			}
 		}
 	})
+	// this test makes sure that Pods with a higher PriorityClass via their PriorityClassName start earlier than Pods with a lower PriorityClassName
+	ginkgo.It("should ensure Pods using a PriorityClassName which is higher begin first", func() {
+		podNamesSeen := []string{}
+		stopCh := make(chan struct{})
+		pods := []v1.Pod{}
+		// define Pod specs - order in from highest to lowest priority
+		ginkgo.By("Defining Pod specs")
+		for i := 4; i >= 1; i-- {
+			pods = append(pods, v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: fmt.Sprintf("priority-class-p%v", i),
+				},
+				Spec: v1.PodSpec{
```

Comment:
The PodSpec doesn't claim any type of resource usage. So they won't test preemption at all.

Reply:
OK, is there a resource which would be recommended to use then?

Reply:
the point here is you need to specify cpu or memory requests. For example: https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/#specify-a-memory-request-and-a-memory-limit